models/engine/db_storage.py

```diff
@@ -88,6 +88,6 @@ def count(self, cls=None):
         """Count objects to class selected or all in storage"""
         new_dict = self.all(cls)
         count = 0
-        for object in new_dict.values():
+        for object_all in new_dict.values():
```

Comment:
It is appreciated the change of name of the variable because it lets what is obtained in that 'for' is the complete object.